AmplifyPlugins/DataStore/Tests/DataStoreHostApp/DataStoreHostApp/AmplifyURLSession.swift

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+
+import Foundation
+import Amplify
+import AWSAPIPlugin
+
+enum AmplifyURLSessionState {
+    case active
+    case inactive
+}
+
+class AmplifyURLSessionNoOperationDataTask: URLSessionDataTaskBehavior {
+    var taskBehaviorIdentifier: Int
+
+    var taskBehaviorResponse: URLResponse?
+
+    init(taskBehaviorIdentifier: Int, taskBehaviorResponse: URLResponse? = nil) {
+        self.taskBehaviorIdentifier = taskBehaviorIdentifier
+        self.taskBehaviorResponse = taskBehaviorResponse
+    }
+
+    static let shared: AmplifyURLSessionNoOperationDataTask =
+        AmplifyURLSessionNoOperationDataTask(taskBehaviorIdentifier: -1)
+
+    func cancel() { }
```

Comment:
nit: Add some logs so that we know when this methods get called. Might help in debugging if the test fails for some reason.